### PR TITLE
Salarycalc 2.0

### PR DESCRIPTION
**Pull request by zupo**
* Automate calculating locationFactors
* Move from cities to countries, And don't normalize against EU-average any more, to keep things simple.
* Get salaries from salary.com.

Ref: https://github.com/niteoweb/operations/issues/1220

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,122 +1,196 @@
...
 
     -   name: Principal Designer
-        baseSalary: 5371
+        baseSalary: 7412
+        salary_com_key: web-designer-iv # TODO web-art-director
 
 
 -   name: Marketing
     roles:
     -   name: Marketing Associate
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Junior Digital Marketer
```

File: config.yml [hunk 2]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 2446
+        baseSalary: 3915
+        salary_com_key: digital-marketing-analyst-i
 
     -   name: Digital Marketing Specialist
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Digital Marketer
```

File: config.yml [hunk 3]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 2955
+        baseSalary: 4797
+        salary_com_key: digital-marketing-analyst-ii
 
     -   name: Senior Digital Marketing Specialist
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Senior Digital Marketer
```

File: config.yml [hunk 4]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 3693
+        baseSalary: 6373
+        salary_com_key: digital-marketing-analyst-iii
 
     -   name: Product Marketing Manager
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Lead Digital Marketer
```

File: config.yml [hunk 5]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 5243
+        baseSalary: 7035
+        salary_com_key: digital-marketing-manager
 
     -   name: Senior Product Marketing Manager
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Principal Digital Marketer
```

File: config.yml [hunk 6]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 6140
+        baseSalary: 11188
+        salary_com_key: digital-marketing-director
 
 
 -   name: Operations
     roles:
     -   name: Customer Support Specialist
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Tech Support I
```

File: config.yml [hunk 7]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 1711
+        baseSalary: 6105
+        salary_com_key: software-engineer-ii
 
     -   name: Customer Support Associate
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Tech Support II
```

File: config.yml [hunk 8]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 1792
+        baseSalary: 5291
+        salary_com_key: technical-support-analyst-ii
 
     -   name: Technical Support Specialist
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Support Manager
```

File: config.yml [hunk 9]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 1979
+        baseSalary: 5840
+        salary_com_key: it-project-manager-i
 
     -   name: Operations Manager
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Project Manager
```

File: config.yml [hunk 10]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 2720
+        baseSalary: 7316
+        salary_com_key: it-project-manager-ii
 
     -   name: Senior Operations Manager
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Operations Manager
```

File: config.yml [hunk 11]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 3903
+        baseSalary: 8796
+        salary_com_key: it-project-manager-iii
 
 
 -   name: Technical
     roles:
     -   name: Junior Programmer
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Junior Engineer
```

File: config.yml [hunk 12]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 2506
+        baseSalary: 4757
+        salary_com_key: software-engineer-i
 
     -   name: Junior Software Engineer
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Engineer
```

File: config.yml [hunk 13]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 3795
+        baseSalary: 6105
+        salary_com_key: software-engineer-ii
 
     -   name: Software Engineer
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Senior Software Engineer
```

File: config.yml [hunk 14]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 4919
+        baseSalary: 7574
+        salary_com_key: software-engineer-iii
 
     -   name: Lead Software Engineer
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Lead Engineer
```

File: config.yml [hunk 15]
```diff
@@ -1,122 +1,196 @@
...
-        baseSalary: 5791
+        baseSalary: 9116
+        salary_com_key: software-engineer-iv
 
     -   name: Principal Software Engineer
```
Comment (dmurko) [inline config.yml]:
```suggestion
    -   name: Principal Engineer
```

**Comment by zupo** [comment]
Went through latest changes with @dmurko, rebased tiny commits into one: https://github.com/niteoweb/salarycalc/pull/44/commits/21db984bf65fd47854a5df611256f75e2bf4f64f

**Comment by zupo** [comment]
> 1. Sync changes of role names.
> 2. The base salaries look off on the first look, against what do I compare them?

1. done
2. You can't, because base salaries are now US averages, no longer discounted by affordability ratio (only location factors are discounted, which is correct).

File: config.yml [hunk 16]
```diff
@@ -1,122 +1,191 @@
...
+        salary_com_key: digital-marketing-director
 
 
 -   name: Operations
     roles:
-    -   name: Customer Support Specialist
-        baseSalary: 1711
+    -   name: Tech Support I
+        baseSalary: 6105
+        salary_com_key: software-engineer-ii
```
Comment (dmurko) [inline config.yml]:
```suggestion
        salary_com_key: technical-support-analyst-i
```

Comment (zupo) [reply config.yml]: Good catch, fixed.

**Comment by dmurko** [comment]
Can we make countries alphabetical?

Screenshot 2020-01-17 at 15 51 40

Other than that, I think we're ready to go.

**Comment by zupo** [comment]
Sorted.

**Pull request merged**